### ISSUE TITLE
Reg loss equals to 0 during the training

### ISSUE THREAD

**Issue by irinushirka** [zylo117/Yet-Another-EfficientDet-Pytorch#611]
Hi guys! I'm training EfficientDet d-3 on COCO-like dataset (9542 train images and 1800 validation images with 6 classes, the input size is 416x416) on Google Colab.

**d2s.yml:**
```
project_name: d2s 
train_set: train2017
val_set: valid2017
num_gpus: 1

mean: [0.485, 0.456, 0.406]
std: [0.229, 0.224, 0.225]

anchors_scales: '[2 ** 0, 2 ** (1.0 / 3.0), 2 ** (2.0 / 3.0)]'
anchors_ratios: '[(1.0, 1.0), (1.4, 0.7), (0.7, 1.4)]'

obj_list: ['beer', 'canned_drink', 'cardboard_tray', 'coca_cola', 'plastic_tray', 'water']
```

Launching the training:
`!python train.py -c 3 -p d2s -n 1 --lr 1e-3 --head_only False --batch_size 1 --num_epochs 5 --load_weights weights/efficientdet-d3.pth --save_interval 100`

The strange thing that I faced during the training - **Reg loss = 0.0000**
![image](https://user-images.githubusercontent.com/57564030/112169879-a373d500-8c03-11eb-9090-23d923d4193f.png)

What does this metric show? What does **Reg loss = 0** mean? Is something wrong with data?

**Comment by zylo117** [comment]
can you show at least one of your images with labels on it?
It's likely that the boxes are too small

**Comment by irinushirka** [comment]
![image](https://user-images.githubusercontent.com/57564030/112325612-e8137500-8cc4-11eb-92b6-cef7b11418f1.png)
![image](https://user-images.githubusercontent.com/57564030/112326229-66701700-8cc5-11eb-8dd4-e5de58ff76c5.png)
Train images examples

**Comment by zylo117** [comment]
Looking good. So it should the problem of annotations.
Does category id starts from 1 instead of 0?

**Comment by irinushirka** [comment]
Yes, sure. Take a look:
![image](https://user-images.githubusercontent.com/57564030/112330703-53f7dc80-8cc9-11eb-96ae-9ce2c70f0161.png)

**Comment by laplaceson** [comment]
Hi , @irinushirka  . Did you solve this problem? Thanks~
